src/SmartTransactionsController.ts

```diff
@@ -255,8 +289,34 @@ export default class SmartTransactionsController extends BaseController<
     return currentIndex === -1 || currentIndex === undefined;
   }
 
-  updateSmartTransaction(smartTransaction: SmartTransaction): void {
-    const { chainId } = this.config;
+  updateSmartTransaction(
+    smartTransaction: SmartTransaction,
+    { networkClientId }: { networkClientId?: NetworkClientId } = {},
+  ) {
+    let { chainId } = this.config;
+    let { ethQuery } = this;
```

Comment:
```suggestion
    let { ethQuery, config: { chainId } = this;
```

ehh idk. `let { ethQuery } = this;` specifically feels funky, but i guess that's only because you aren't usually deconstructing `this`

Reply:
maybe even `let chainId, ethQuery;` since you default it in `#updateSmartTransaction` already

Reply:
Its kindof annoying but if I don't add the defaults here and then pass then both as undefined it will override the defaults in `#updateSmartTransaction`.